pkg/proc/native/support_sentinel_windows.go

```diff
@@ -1,8 +1,6 @@
 // This file is used to detect build on unsupported GOOS/GOARCH combinations.
 
-//go:build windows && !amd64 && !(arm64 && exp.winarm64)
-// +build windows
-// +build !amd64
-// +build !arm64 !exp.winarm64
+//go:build windows && !amd64 && !arm64
```

Comment:
I ran the tests 10 times, of these 3/10 got stuck on TestStepConcurrentDirect. Therefore I think the build guard should stay and TestStepConcurrentDirect should have a skipOn line.

Reply:
Haven't found this failure mode in my runs. Where are you running the tests? I don't see the failures in the teamcity dashboard (I'm interested in the logs).

Reply:
Teamcity, you don't see them on the dashboard because they are personal builds:

https://delve.teamcity.com/buildConfiguration/Delve_windows_arm64_1_19/41584
https://delve.teamcity.com/buildConfiguration/Delve_windows_arm64_1_19/41587
https://delve.teamcity.com/buildConfiguration/Delve_windows_arm64_1_19/41588

I think you can see them if you go on the build history of your builder (unless teamcity is only showing them to me because they are mine).

Reply:
Thanks, I can see them. Will take a look before skipping the test.

Reply:
Ok, I have no clue why TestStepConcurrentDirect is failing and won't have time during the following weeks to look into it, so I'll skip it for now and keep the `exp.winarm64` gate.